src/test/java/net/snowflake/client/core/arrow/TwoFieldStructToTimestampTZConverterTest.java

```diff
@@ -127,6 +129,11 @@ public void testTimestampTZ() throws SFException {
       Date date = converter.toDate(j, getTimeZone(), false);
       Time time = converter.toTime(j);
       String tsStr = converter.toString(j);
+      if (tsStr != null) {
+        assertFalse(converter.isNull(j));
+      } else {
+        assertTrue(converter.isNull(j));
+      }
```

Comment:
Without the fix, `IndexOutOfRangeException` is raised, because the base class's VectorValue is not initialized properly.